rotkehlchen/chain/ethereum/modules/curve/decoder.py

```diff
@@ -461,6 +607,32 @@ def _maybe_enrich_curve_transfers(self, context: EnricherContext) -> TransferEnr
             context.event.counterparty = CPT_CURVE
         return DEFAULT_ENRICHMENT_OUTPUT
 
+    @staticmethod
+    def _sort_events(
+            action_type: Literal['addition', 'removal'],
+            return_or_receive_event: 'EvmEvent',
+            withdrawal_or_deposit_events: list['EvmEvent'],
+            all_events: list['EvmEvent'],
+    ) -> list['EvmEvent']:
+        """
+        This method sorts the events in the expected order and adds
+        extra info to the in/out event depending on the action_type.
+        """
+        if action_type == 'addition':
+            return_or_receive_event.extra_data = {
+                'deposit_events_num': len(withdrawal_or_deposit_events),
+            }
+        else:  # can only removal
```

Comment:
I already asked you to fix it and you marked this as resolved but did not do it ... 3 weeks ago!!!

https://github.com/rotki/rotki/pull/5736/files#r1139452060
```suggestion
        else:  # can only be removal
```